navigator_helpers/tasks/evaluation/create_pdf_report.ipynb

```diff
@@ -0,0 +1,826 @@
+    "    ax2.set_ylabel('Cumulative Percentage', fontsize=10, color='#1D0B32')\n",
+    "    ax2.set_ylim([0, 110])\n",
+    "    \n",
+    "    ax1.tick_params(axis='both', colors='#1D0B32')\n",
+    "    ax2.tick_params(axis='both', colors='#1D0B32')\n",
+    "    ax1.tick_params(axis='y', labelsize=6)\n",
+    "    ax2.tick_params(axis='y', labelsize=6)\n",
+    "    \n",
+    "    ax1.set_xticks(range(len(data)))\n",
+    "    truncated_labels = [str(label)[:25] + ('...' if len(str(label)) > 25 else '') for label in data.index]\n",
+    "    ax1.set_xticklabels(truncated_labels, rotation=45, ha='right', fontsize=6, color='#1D0B32')\n",
+    "    \n",
+    "    for i, v in enumerate(data['count']):\n",
+    "        ax1.text(i, v, f'{v:.2f}' if isinstance(v, float) else f'{v}', ha='center', va='bottom', fontsize=8, color='#1D0B32')\n",
+    "    \n",
+    "    ax2.yaxis.set_major_formatter(plt.FuncFormatter(lambda y, _: f'{y:.0f}%'))\n",
+    "    \n",
+    "    plt.tight_layout()\n",
+    "    img_buffer = io.BytesIO()\n",
+    "    plt.savefig(img_buffer, format='png', dpi=300, bbox_inches='tight')\n",
+    "    img_buffer.seek(0)\n",
+    "    return Image(img_buffer, width=7*inch, height=4.5*inch)\n",
+    "\n",
+    "def create_text_diversity_chart(text_diversity_df: pd.DataFrame) -> Image:\n",
+    "    plt.figure(figsize=(FIG_WIDTH, FIG_HEIGHT))\n",
+    "    ax = sns.barplot(x=text_diversity_df.index, y='diversity_index', data=text_diversity_df, color='#4F00A9')\n",
+    "    ax.set_facecolor('white')\n",
+    "    plt.gcf().patch.set_facecolor('white')\n",
+    "    \n",
+    "    plt.title(\"Text Diversity Indices\", fontsize=10, color='#1D0B32')\n",
+    "    plt.ylabel(\"Diversity Index\", fontsize=10, color='#1D0B32')\n",
+    "    plt.xlabel(\"\", fontsize=10, color='#1D0B32')\n",
+    "    plt.xticks(rotation=45, ha='right', fontsize=8, color='#1D0B32')\n",
+    "    \n",
+    "    plt.ylim(0, 1)\n",
+    "    \n",
+    "    ax.tick_params(axis='both', colors='#1D0B32')\n",
+    "    ax.tick_params(axis='y', labelsize=6)\n",
+    "    \n",
+    "    for i, v in enumerate(text_diversity_df['diversity_index']):\n",
+    "        ax.text(i, v, f'{v:.2f}', ha='center', va='bottom', fontsize=8, color='#1D0B32')\n",
+    "    plt.tight_layout()\n",
+    "    \n",
+    "    img_buffer = io.BytesIO()\n",
+    "    plt.savefig(img_buffer, format='png', dpi=300, bbox_inches='tight')\n",
+    "    img_buffer.seek(0)\n",
+    "    return Image(img_buffer, width=7*inch, height=4.5*inch)\n",
+    "\n",
+    "def create_schema_table(dataset_df: pd.DataFrame, data: Dict[str, Any]) -> Tuple[Table, Dict[str, float]]:\n",
+    "    schema_data = [['Column Name', 'Data Type', 'Total Count', '% Null', 'Average Length', 'Avg Tokens']]\n",
+    "    avg_lengths = {}\n",
+    "    for col in dataset_df.columns:\n",
+    "        dtype = str(dataset_df[col].dtype)\n",
```

Comment:
I suggest that we use the column data types determined [here](https://github.com/gretelai/navigator-helpers/blob/f18f2073f77050c1faec0aee701787084f4449e2/navigator_helpers/tasks/evaluation/evaluation.py#L47) (same as my comment on the distribution plots). The same suggestion goes towards the column counts in the "Dataset Overview" table above